.github/workflows/prerelease_staged_publish.yml

```diff
@@ -73,6 +73,52 @@ jobs:
           gpg_passphrase: ${{ env.GPG_PASSPHRASE }}
           gpg_private_key_base64: ${{ env.GPG_PRIVATE_KEY_BASE64 }}
 
+  publishing-to-s3-macos:
```

Comment:
This will be not needed if we create different suffix for macos. We could re-use previous job with 1 more `assetsType`